### PR TITLE
`impl Clone`

### PR DESCRIPTION
**Pull request by dpc**
Start work on #4

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -106,6 +110,13 @@ impl<T, H: ExceptionHandler> DangerousOption<T, H> {
     }
 }
 
+#[cfg(feature = "std")]
```
Comment (Kixunil) [inline src/lib.rs]: No need for this. `core` has `Clone` too.

File: src/lib.rs [hunk 2]
```diff
@@ -106,6 +110,13 @@ impl<T, H: ExceptionHandler> DangerousOption<T, H> {
...
+impl<T : Clone, H: ExceptionHandler> std::clone::Clone for DangerousOption<T, H> {
```
Comment (Kixunil) [inline src/lib.rs]: Why not use `#[derive(Clone)]`?

**Review by Kixunil — CHANGES_REQUESTED** [review]
Thank you for PR!

Please remove dependency on `std`. Use `#[derive()]` unless you have a reason not to.

**Comment by dpc** [comment]
How about now? Can't use `derive` as phanton type does not impl `Clone`.

**Review by Kixunil — APPROVED** [review]
OK, thanks!

**Pull request merged**